packages/runtime/container-runtime/src/blobManager/blobManager.ts

```diff
@@ -435,6 +452,15 @@ export class BlobManager {
 		if (this.runtime.attachState === AttachState.Detached) {
 			return this.createBlobDetached(blob);
 		}
+		return this.mc.config.getBoolean("Fluid.Runtime.UploadBlobPlaceholders") === true
```

Comment:
what is the test strategy for this? currently its not enabled for any tests.

Reply:
Yeah I need to figure that out still.  Haven't looked at how other feature flags do it yet, but seems like it would be a waste of time to test both on/off for most of our tests.  At first glance looks like the blob and GC tests are the main interesting ones.

Reply:
we will have production users on both behaviors, seem like we need strong guarantees they both continue to work until we only have one behavior.